admin_guide/managing_networking.adoc

```diff
@@ -1184,6 +1184,11 @@ resources can recognize the traffic.
 Unlike the egress router, this is subject to `EgressNetworkPolicy` firewall
 rules.
 
+[NOTE]
+====
+Assigning static IPs addresses for projects in your cluster requires the SDN to use either the *ovs-networkpolicy* or *ovs-multitenant* network plug-ins.
```

Comment:
The package names need to be in [backticks](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).

Also, where do we talk about these packages? I don't either of those names in the master branch.